vendor/github.com/imdario/mergo/merge.go

```diff
@@ -39,6 +41,8 @@ type Transformers interface {
 // short circuiting on recursive types.
 func deepMerge(dst, src reflect.Value, visited map[uintptr]*visit, depth int, config *Config) (err error) {
 	overwrite := config.Overwrite
+	overwriteWithEmptySrc := config.overwriteWithEmptyValue
+	config.overwriteWithEmptyValue = false
```

Comment:
this change looks buggy... opened https://github.com/imdario/mergo/issues/187

Reply:
Rolled back to mergo v0.3.5, so no change in k8s mergo deps for now.